database/redis/src/main/java/com/example/javaopencommerce/order/RedisCardRepositoryImpl.java

```diff
@@ -4,10 +4,12 @@
 import io.quarkus.redis.datasource.list.ListCommands;
 import lombok.extern.log4j.Log4j2;
 
+import javax.enterprise.context.ApplicationScoped;
 import java.util.Collections;
 import java.util.List;
 
 @Log4j2
+@ApplicationScoped
```

Comment:
Not a fan of that annotation. It would be better to create bean with a @Producer.